### DISCUSSION TITLE
Exclude resources from validation?

### DISCUSSION THREAD

**Discussion by BenPortner** [frictionlessdata/frictionlessdata.io#587] [Q&A] [answered]
Dear frictionless data community,

I am currently realizing my first frictionless project and I came across a problem:
My datapackage consists of a bunch of raw data files in `./data/raw` and a bunch of processed files in `./data`. I would like to include both raw data and processed data in the `datapackage.json` for completeness and documentation purpose. However, when I do so validation fails for obvious reasons (the raw data is in bad shape hence the processing). So here is my question: Is it possible to set a flag in the resource definition in `datapackage.json` to indicate that the corresponding resource should be skipped during datapackage validation? 

Any help is appreciated :)

All the best.
Ben

**Comment by rufuspollock** [comment]
@BenPortner good question. Referring here to @roll who would be expert re the validation side of things.

**Comment by roll** [comment, marked as answer]
Hi @BenPortner,

I've created a feature request for this use case - https://github.com/frictionlessdata/frictionless-py/issues/645

**Comment by BenPortner** [comment]
Thanks, @roll. I'm looking forward to its implementation 😃
